symengine/tests/test_series_expansion.py

```diff
@@ -1,22 +1,23 @@
 from symengine.utilities import raises
 from symengine.lib.symengine_wrapper import (series, have_piranha, have_flint,
-    Symbol, Integer, sin, cos, exp, sqrt, E)
+                                             Symbol, Integer, sin, cos, exp,
+                                             sqrt, E)
```

Comment:
Is this enforced in PEP 8 ? I like the previous one better, as it has less line.